SkeinforgeMe.py

```diff
@@ -21,7 +21,7 @@
 # THEN you are ready to run this from the command line. It will skein EVERY .stl or .STL in the Prusa Mendel 
 # by walking all directory trees.
 # best to set this up at night. You wake up to a directory full of .gcode files. =]
-
+#added a comment - PC
```

Comment:
Looks good.
Just one minor thing. Needs space before comment to match the format.